engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/compile/PartSubGraphCompiler.scala

```diff
@@ -273,3 +297,21 @@ class PartSubGraphCompiler(protected val classLoader: ClassLoader,
     }
   }
 }
+
+case class ExpressionTypingResult(typingResult: TypingResult, typingInfo: Option[ExpressionTypingInfo]) {
```

Comment:
can they be private?

Reply:
Yep, I'll add it to companion object with private access